helix.go

```diff
@@ -301,7 +307,10 @@ func (c *Client) getBaseURL(path string) string {
 		}
 	}
 
-	return APIBaseURL
+	if c.opts.APIBaseURL == "" {
```

Comment:
This `if` condition is redundant. The return statement you've introduced here is sufficient.